scienceworld/scienceworld.py

```diff
@@ -139,11 +134,11 @@ def tasks(self):
     @property
     def task_names(self):
         """ Get the name for the supported tasks in ScienceWorld. """
-        return list(ID2TASK.values())
+        return sorted(ID2TASK.values())
```

Comment:
Why the `sorted` here?

Reply:
Ok, I see later with the unit test, it is to make sure Python and Scala (i.e., getTaskNames) returns the same thing. I think I'd prefer the tasks to be listed as they appear in the paper. That's why I was using an OrderedDict for ID2Task.

Reply:
@MarcCote, that makes sense! I've moved the sorting to the test. I briefly tried to have the Scala code output the same order, but it seemed like more effort than it was worth.